templates_jinja2/avatars2018.html

```diff
@@ -7,13 +7,13 @@
 {% block content %}
 <div class="container">
   <div class="row">
-    <div class="col-lg-8 col-lg-offset-2">
+    <div class="col-lg-8 col-lg-offset-2 avatar-container">
```

Comment:
I believe bootstrap has a built in css property called `text-center`

Reply:
Oh, good call.